### DISCUSSION TITLE
Simulating Recharge Due to Irrigation

### DISCUSSION THREAD

**Discussion by egwsim** [pastas/pastas#792] [Q&A] [answered]
I've been trying to use Pastas to simulate a system of wells and one of the primary sources of recharge is irrigation. The main input for data that I have was primarily used in another groundwater model where irrigation is assumed to be input directly into the water table, skipping any effect of unsaturated non-linear recharge. What is the best way to do this in Pastas? I know you can change the rfunc in the StressModel to Exponential which doesn't have any delay. Is this the best way? Since this is a very common scenario for the types of projects I want to use Pastas for, it would great to figure out what the best way to do this is.

Thanks!

**Comment by dbrakenhoff** [comment, marked as answer]
Hi @egwsim, you're spot on, using the `ps.Exponential` response function will cause the irrigation to lead to an immediate rise of the water table, that (slowly) subsides over time. For an even more extreme effect (but probably not very realistic) you can use the `ps.One` response function, that simply directly transfers the irrigation to fluctuations of the water table through some factor.
